sysbuild/Kconfig.secureboot

```diff
@@ -37,6 +37,7 @@ config SECURE_BOOT_APPCORE_SUPPORTED_HASH_NONE
 config SECURE_BOOT_APPCORE_SUPPORTED_SIGNATURE_HARDWARE
 	bool
 	default y if SECURE_BOOT_SIGNATURE_TYPE_ECDSA && (SOC_SERIES_NRF91X || SOC_NRF52840)
+	default y if SECURE_BOOT_SIGNATURE_TYPE_ED25519 && SOC_NRF54L15_CPUAPP
```

Comment:
extend for enabling this on SOC_NRF54l05_CPUAPP and SOC_NRF54l10_CPUAPP
same in else places in that PR